src/pages/atomicdex/api/v20/orderbook/index.mdx

```diff
@@ -0,0 +1,359 @@
+| is\_mine                          | bool              | Whether the order is placed by me                                                                                                 |
+| conf_settings                     | object            | A standard `ConfSettings` object                                                                                                  |
+| base_max_volume                   | object            | A standard `OrderAmount` object representing the maximum base amount for this order                                               |
+| base_min_volume                   | object            | A standard `OrderAmount` object representing the minimum base amount for this order                                               |
+| rel_max_volume                    | object            | A standard `OrderAmount` object representing the maximum rel amount for this order                                                |
+| rel_min_volume                    | object            | A standard `OrderAmount` object representing the minimum rel amount for this order                                                |
+| base_max_volume_aggr              | object            | A standard `OrderAmount` object representing the maximum aggregated base amount at the specified price. The sum of max base volumes of the current order and all orders with a worse price. |
+| rel_max_volume_aggr               | object            | A standard `OrderAmount` object representing the maximum aggregated rel amount at the specified price. The sum of max rel volumes of the current order and all orders with a worse price. |
+
+
+**`ConfSettings` object structure**
+
+| Structure                         | Type              | Description                                                                                                                       |
+| --------------------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| base\_confs                       | number            | the confirmations settings of `base` coin set by the offer provider                                                               |
+| base\_nota                        | bool              | the notarisation settings of `base` coin set by the offer provider                                                                |
+| rel\_confs                        | number            | the confirmations settings of `rel` coin set by the offer provider                                                                |
+| rel\_nota                         | bool              | the notarisation settings of `rel` coin set by the offer provider                                                                 |
+
+
+#### 📌 Examples
+
+#### Command
+
+<CodeGroup title="" tag="POST" label="orderbook">
```

Comment:
hey, I updated this component to work like so:
```
<CodeGroup title="Generate Invoice" tag="POST" label="generate_invoice" mm2MethodDecorate="true">
```
instead of `{{ mm2MethodDecorate : true }}`

see: https://github.com/KomodoPlatform/komodo-docs-mdx/blob/dev/STYLE_GUIDE.md#:~:text=A%20working%20code%20sample%20would%20look%20like%20this%3A